qf_lib/data_providers/bloomberg_beap_hapi/bloomberg_beap_hapi_request_provider.py

```diff
@@ -87,7 +91,8 @@ def create_request(self, request_id: str, universe_url: str, fieldlist_url: str)
             },
             'pricingSourceOptions': {
                 '@type': 'DataPricingSourceOptions',
-                'prefer': {'mnemonic': 'BGN'}
+                'exclusive': True,
+                'prefer': {'mnemonic': pricing_source or 'BGN'}
```

Comment:
The default parameter in `get_current_values`, `get_history`, and `create_request` all have `None` as the default pricing parameter which then turns into `BGN` here. I feel it's more transparent to set the default parameter for `pricing_source` to `BGN` for all the functions.